tests/suites.c

```diff
@@ -179,6 +179,7 @@ static int execute_test_case(int svr_argc, char** svr_argv,
     THREAD_TYPE serverThread;
     char        commandLine[MAX_COMMAND_SZ];
     char        cipherSuite[MAX_SUITE_SZ+1];
+    char        portNumber[8];
```

Comment:
fsanitizer reported use of variable declared in local scope after scope had ended.